redaxo/src/core/pages/system.clangs.php

```diff
@@ -98,25 +99,42 @@
                     <th class="rex-table-id">' . rex_i18n::msg('id') . '</th>
                     <th>' . rex_i18n::msg('clang_code') . '</th>
                     <th>' . rex_i18n::msg('clang_name') . '</th>
+                    <th>' . rex_i18n::msg('clang_locale') . '</th>
                     <th class="rex-table-priority">' . rex_i18n::msg('clang_priority') . '</th>
                     <th class="rex-table-action" colspan="3">' . rex_i18n::msg('clang_function') . '</th>
                 </tr>
             </thead>
             <tbody>
     ';
 
+
 // Add form
 if ('addclang' == $func) {
     // ----- EXTENSION POINT
     $metaButtons = rex_extension::registerPoint(new rex_extension_point('CLANG_FORM_BUTTONS', ''));
 
+    $localeFormElement = '<input class="form-control" type="text" id="rex-form-clang-locale" name="clang_locale" value="' . rex_escape($clang_locale) . '" />';
+
+    $locales = explode("\n", trim(shell_exec('locale -a')));
```

Comment:
das wird vermutlich in sehr vielen servern nicht funktionieren.

sollten wir stattdessn ggf. mit symfony/intl arbeiten?
https://symfony.com/doc/current/components/intl.html

Reply:
Du meinst wegen `shell_exec` oder das wir die `intl` Erweiterung als Abhängigkeit aufnehmen sollten?

Reply:
ich vermute es wird nicht funktionieren wg. `shell_exec` - ja.

ob man für symfony/intl die Intl extension brauch, weiß ich nicht

Reply:
> ob man für symfony/intl die Intl extension brauch, weiß ich nicht

Benötigt man. Zitat von der Symfony Seite

> The replacement layer is limited to the locale "en". If you want to use other locales, you should install the intl extension instead.


> ich vermute es wird nicht funktionieren wg. shell_exec - ja.

Vielleicht kann @ruedigernitzsche hierzu was sagen, wie die Chancen stehen, dass es auf den Servern vorhanden ist.

Reply:
So wie es aktuell angedacht ist, dass man die exakte Locale angibt, also sowas wie `de_de.utf8` etc., würden die intl locales uns wahrscheinlich nicht helfen.